spaces/S000015/properties/P000110.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000015
+property: P000110
+value: true
+---
+
+Let $\mathcal{U}_n = \{\{m\}\cup\{k\in \omega : k\geq n\}:m\in \omega\}$, then $\text{St}(x, \mathcal{U}_n)= \begin{cases} \omega, & x\geq n \\ \{x\}\cup \{k : k\geq n\}, & x < n \end{cases}$ so that $(\text{St}(x, \mathcal{U}_n))_{n\geq 0}$ is a local base at $x\in \omega$.
```

Comment:
```suggestion
Let $\mathcal{U}_n = \{\{m\}\cup\{k\in \omega : k\geq n\}:m\in \omega\}$, which is an open cover of $X$. Then $\text{St}(x, \mathcal{U}_n)= \begin{cases} \omega, & \text{if }x\geq n \\ \{x\}\cup \{k : k\geq n\}, & \text{if }x < n \end{cases}$, so that $(\text{St}(x, \mathcal{U}_n))_{n\geq 0}$ is a local base at $x\in \omega$.
```